terraform/modules/prom-ec2/alerts-config/alerts/observe-alerts.yml

```diff
@@ -30,10 +30,21 @@ groups:
         product: "prometheus"
         severity: "page"
     annotations:
-        summary: "There is one or less Prometheis that can be scraped"
+        summary: "There is one or fewer Prometheis that can be scraped"
         logs: "https://kibana.logit.io/s/8fd50110-7b0c-490a-bedf-7544daebbec4/app/kibana#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:now-15m,mode:quick,to:now))&_a=(columns:!(_source),index:'*-*',interval:auto,query:(query_string:(query:'tags:%20prometheus')),sort:!('@timestamp',desc))"
         runbook: "https://re-team-manual.cloudapps.digital/prometheus-for-gds-paas-users.html#re-observe-prometheus-below-threshold"
 
+  - alert: RE_Observe_Prometheus_AtLeastOneMissing
+    expr: sum(up{job="prometheus"}) < 3
+    for: 3m
+    labels:
+        product: "prometheus"
+        severity: "ticket"
+    annotations:
+        summary: "At least one Prometheus can't be scraped"
+        logs: "https://kibana.logit.io/s/8fd50110-7b0c-490a-bedf-7544daebbec4/app/kibana#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:now-15m,mode:quick,to:now))&_a=(columns:!(_source),index:'*-*',interval:auto,query:(query_string:(query:'tags:%20prometheus')),sort:!('@timestamp',desc))"
+        runbook: "https://re-team-manual.cloudapps.digital/prometheus-for-gds-paas-users.html#re-observe-prometheus-at-leat-one-missing"
```

Comment:
```suggestion
        runbook: "https://re-team-manual.cloudapps.digital/prometheus-for-gds-paas-users.html#re-observe-prometheus-at-least-one-missing"
```

Reply:
God damn it. Also, of course, this section doesn't exist yet so that's another TODO